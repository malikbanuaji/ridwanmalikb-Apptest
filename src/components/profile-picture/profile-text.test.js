import * as React from 'react';
import ProfileText from './profile-text';

import {render} from '@testing-library/react-native';

test('should render properly', () => {
  const component = render(<ProfileText firstName="" lastName="" />);
  expect(component.toJSON()).toMatchSnapshot();
});

test('should render properly wiht names', () => {
  const component = render(
    <ProfileText firstName="Malik" lastName="Banuaji" />,
  );
  expect(component.toJSON()).toMatchSnapshot();
});
