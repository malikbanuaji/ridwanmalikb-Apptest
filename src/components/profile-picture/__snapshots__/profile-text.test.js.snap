// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render properly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#99aab5",
      "borderRadius": 8,
      "height": 42,
      "justifyContent": "center",
      "width": 42,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#ffffff",
        "fontSize": undefined,
        "fontWeight": "500",
      }
    }
  />
</View>
`;

exports[`should render properly wiht names 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#99aab5",
      "borderRadius": 8,
      "height": 42,
      "justifyContent": "center",
      "width": 42,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#ffffff",
        "fontSize": undefined,
        "fontWeight": "500",
      }
    }
  >
    M
    B
  </Text>
</View>
`;
